### PR TITLE
feat(tabview): add tab text font size property

### PR DESCRIPTION
**Pull request by MartoYankov**
Allow setting tab titles font size without changing the font size of all contents of the tab. The basic font-size css property is inherited and changes the contents of the tabs.

### CODE REVIEW

**Comment by pedromorgan** [comment]
LGTM and kinda what my problem is atmo

**Comment by MartoYankov** [comment]
test branch_widgets#myankov/tab-font-size

**Comment by SvetoslavTsenov** [comment]
test branch_widgets#myankov/tab-font-size

**Comment by SvetoslavTsenov** [comment]
run ci

File: tns-core-modules/ui/tab-view/tab-view.ios.ts [hunk 1]
```diff
@@ -448,6 +448,13 @@ export class TabView extends TabViewBase {
         selectedIndexProperty.coerce(this);
     }
 
+    [tabTextFontSizeProperty.getDefault](): { nativeSize: number } {
```
Comment (manoldonev) [inline tns-core-modules/ui/tab-view/tab-view.ios.ts]: Shouldn't return type be just `number` here? (based on the d.ts)

**Comment by MartoYankov** [comment]
test branch_widgets#myankov/tab-font-size

**Pull request merged**